<html>
<head>
	<link rel="Stylesheet" type="text/css" href="../../Public/templates/skinny_disqus.css" />
	<title>A simple TeX example</title>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
</head>

<body>
<div class="container">
    <div class="header">
		<h1>A simple TeX example</h1> 
	</div><!-- end of header -->
	<div class="post">
	
<p>
<a href="Category_LaTeX.html">Category:LaTeX</a>
&lt;latex&gt;
% Augix' LaTeX template
\documentclass[11pt]{article}
\usepackage[top=3cm, bottom=3cm, left=3cm, right=3cm]{geometry}
\usepackage{pgf}
</p>

<p>
\author{Augix G.H. XU\\
CAS-MPG Partner Institute for Computational Biology \\
Max Planck Institute for Evolutionary Anthropology
}
\date{Dec 4, 2007}
\title{Thesis Proposal: \\COMPARATIVE ANALYSIS OF PRIMATE ALTERNATIVE SPLICING USING EXON ARRAYS}
</p>

<p>
\begin{document}
</p>

<p>
\pagestyle{empty}
</p>

<p>
%\nocite{*}	%The \nocite command produces no text, but writes key_list, which is a list of one or more citation keys, on the aux file. It can help us to find the references which are not cited.
</p>

<p>
\maketitle
</p>

<p>
%\hfill  \today \\
</p>


<p>
The use of alternatively spliced transcript forms among different human tissues is well documented and is estimated to affect 40 - 60\% of all human genes\cite{Johnson:2003}\cite{Kan:2001}\cite{Mironov:1999}\cite{Modrek:2001wd}\cite{Modrek:2002cr} and 74\% of multiexon human genes\cite{Johnson:2003}. Differences in splicing between species, however, are still poorly understood. These differences may include not only conservation of alternative splicing between species, but also differences in splicing of transcripts constitutively spliced within each species. 
</p>

<p>
In this study, we address both conservation of the alternative splicing between closely related primate species (humans, chimpanzees, orangutans and rhesus macaques), as well as the splicing differences in between the species by measuring expression profiles in four tissues (cortex, cerebellum, liver and testis) using Affymetrix Human Exon Arrays. 
</p>

<p>
In agreement with previous estimates, about 70\% of all expressed multiexon genes show evidence for alternative splicing among different tissues in humans. This is also the case for chimpanzees. Surprisingly, approximately 15\% of genes are spliced differently in between the species. These differences are not distributed equally among tissues, with testis having the most differences and the two brain tissues the least (Figure~\ref{fig:DS}). Furthermore, our results suggest that less than 3\% of genes show species-specific alternative splicing. 
</p>

<p>
These results allow us, for the first time, to estimate the extent of splicing differences among closely related species and compare it to the patterns of alternative splicing observed among tissues. Further, using orangutan and rhesus macaque data, we are able to identify splicing differences specific to humans. Given the surprisingly high proportion of differential splicing observed between the species, our results may imply that changes in splicing have contributed substantially to the evolution of human species. 
</p>

<p>
 \vfill
</p>

<p>
% Put graphics
\begin{figure}
  \begin{center}
</p>
<blockquote>
\label{fig:DS}
\caption{Number of differently spliced genes}
\includegraphics[scale=0.3]{f1.jpg}
\item{Numbers of genes showing evidence of RNA
splicing differences in between human and
chimpanzee in 4 different tissues (cortex,
cerebellum, liver, testis).}
</blockquote>
<p>
  \end{center}
\end{figure}
</p>



<p>
\bibliographystyle{amsplain}
\bibliography{AS}
</p>

<p>
\end{document}
&lt;/latex&gt;
</p>

	</div><!-- end of post -->
<br>

</div><!-- end of container -->

</body>
</html>
