<html>
<head>
	<title>Bioinformatics</title>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />

	<link rel="Stylesheet" type="text/css" href="../templates/skinny_disqus.css" />

    <!-- SyntaxHighlighter start -->
    <link type="text/css" rel="stylesheet" href="../templates/SyntaxHighlighter/styles/shCore.css" /> 
    <link type="text/css" rel="stylesheet" href="../templates/SyntaxHighlighter/styles/shThemeDefault.css" /> 
    <script type="text/javascript" src="../templates/SyntaxHighlighter/scripts/shCore.js"></script> 
    <script type="text/javascript" src="../templates/SyntaxHighlighter/scripts/shBrushBash.js"></script>
    <script type="text/javascript" src="../templates/SyntaxHighlighter/scripts/shBrushJava.js"></script>
    <script type="text/javascript" src="../templates/SyntaxHighlighter/scripts/shBrushPerl.js"></script>
    <script type="text/javascript" src="../templates/SyntaxHighlighter/scripts/shBrushPhp.js"></script>
    <script type="text/javascript" src="../templates/SyntaxHighlighter/scripts/shBrushPython.js"></script>
    <script type="text/javascript" src="../templates/SyntaxHighlighter/scripts/shBrushR.js"></script>
    <script type="text/javascript" src="../templates/SyntaxHighlighter/scripts/shBrushPlain.js"></script>
    <script type="text/javascript" src="../templates/SyntaxHighlighter/scripts/shBrushXml.js"></script>
    <script type="text/javascript">
      SyntaxHighlighter.all();
    </script> 
    <!-- SyntaxHighlighter end -->

    <!-- Google Analytics -->
    <script type="text/javascript">
      var _gaq = _gaq || [];
      _gaq.push(['_setAccount', 'UA-27503060-1']);
      _gaq.push(['_trackPageview']);
      (function() {
        var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
        ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
        var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
      })();
    </script>
    <!-- Google Analytics end -->

</head>

<body>
<a name='top'></a>
<div class="container">
    <div class="header">
		<h1>Bioinformatics</h1> 
	</div><!-- end of header -->
	<div class="post"><!-- start of post -->
	
<h1>Table of Content</h1>
<div class="toc">
<ul>
<li><a href="#toc_1">Databases</a>
<ul>
<li><a href="#toc_1.1">Genome Browser</a>
<li><a href="#toc_1.2">Ensembl</a>
<ul>
<li><a href="#toc_1.2.1">biomart</a>
<ul>
<li><a href="#toc_1.2.1.1">R package</a>
</ul>
<li><a href="#toc_1.2.2">genebuild</a>
<li><a href="#toc_1.2.3">Regulatory Build</a>
</ul>
<li><a href="#toc_1.3">VEGA database</a>
<li><a href="#toc_1.4">Consensus CDS (CCDS) project</a>
</ul>
<li><a href="#toc_2">Applications</a>
<ul>
<li><a href="#toc_2.1">Homology search</a>
<li><a href="#toc_2.2">Next-Gen Sequencing</a>
<li><a href="#toc_2.3">Computing</a>
<li><a href="#toc_2.4">Phylogenetic tree</a>
<ul>
<li><a href="#toc_2.4.1">R package: phangorn</a>
</ul>
<li><a href="#toc_2.5">Other</a>
</ul>
<li><a href="#toc_3">Topic</a>
</ul>
</div>

<h1 id="toc_1">Databases</h1>
<h2 id="toc_1.1">Genome Browser</h2>
<p>
<a href="http://genome.ucsc.edu/">http://genome.ucsc.edu/</a>
</p>

<p>
length information of human chromosomes
</p>

<p>
<a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgsid=1230&amp;chromInfoPage=">http://genome.ucsc.edu/cgi-bin/hgTracks?hgsid=1230&amp;chromInfoPage=</a>
</p>

<h2 id="toc_1.2">Ensembl</h2>
<h3 id="toc_1.2.1">biomart</h3>
<p>
Automatic fetching of gene annotation, structure, function. Comparative genomics analysis.
</p>

<h4 id="toc_1.2.1.1">R package</h4>
<pre>
mart = useMart("ensembl", dataset="hsapiens_gene_ensembl")
getGene(id = 'CD44', type = "hgnc_symbol", mart = mart)
</pre>

<h3 id="toc_1.2.2">genebuild</h3>

<p>
<a href="http://www.ensembl.org/info/docs/genebuild/genome_annotation.html">reference</a>
</p>

<p>
Ensembl transcripts displayed on our website are products of the Ensembl automatic pipeline, termed the Ensembl genebuild.
</p>

<p>
All Ensembl transcripts are based on experimental evidence and thus the automated pipeline relies on the mRNAs and protein sequences deposited into public databases from the scientific community.
</p>

<p>
Exonerate is a tool for pairwise sequence comparison, used in the Ensembl genebuild to align mRNA to the assembly.
</p>

<h3 id="toc_1.2.3">Regulatory Build</h3>

<p>
Promoters are not available for most genes in Ensembl.
</p>

<p>
One can find information on genome-wide studies of regulatory regions in Ensembl Regulatory Build.
</p>

<h2 id="toc_1.3">VEGA database</h2>
<p>
The Vertebrate Genome Annotation (VEGA) database is a central repository for high quality manual annotation of vertebrate finished genome sequence. 
</p>

<p>
Human, mouse and zebrafish are in the process of being completely annotated.
</p>

<p>
In other species, the annotation is only of specific genomic regions of particular biological interest.
</p>

<h2 id="toc_1.4">Consensus CDS (CCDS) project</h2>

<p>
The CCDS project is a collaborative effort to identify a core set of protein coding regions that are consistently annotated and of high quality. 
</p>

<p>
Initial results from the Consensus CDS (CCDS) project are now available through the appropriate Ensembl gene pages and from the CCDS project page at NCBI.
</p>


<h1 id="toc_2">Applications</h1>
<h2 id="toc_2.1">Homology search</h2>
<ul>
<li>
<a href="BLAST and formatdb.html">BLAST and formatdb</a>

<li>
<a href="Blat.html">Blat</a>

<li>
<a href="liftOver.html">liftOver</a>

</ul>
 
<h2 id="toc_2.2">Next-Gen Sequencing</h2>
<ul>
<li>
<a href="Tophat.html">Tophat</a>

<li>
<a href="SAMTools.html">SAMTools</a>

<li>
<a href="Pysam.html">Pysam</a>

</ul>

<h2 id="toc_2.3">Computing</h2>
<ul>
<li>
<a href="SGE in eva cluster.html">SGE in eva cluster</a>

<li>
<a href="SGE in PICB cluster.html">SGE in PICB cluster</a>

</ul>

<h2 id="toc_2.4">Phylogenetic tree</h2>
<h3 id="toc_2.4.1">R package: phangorn</h3>
<ul>
<li>
<a href="http://cran.r-project.org/web/packages/phangorn/index.html">Souce code</a>

<li>
Estimating phylogenetic trees with phangorn <a href="http://cran.univ-lyon1.fr/web/packages/phangorn/vignettes/Trees.pdf">PDF</a>

<li>
Special features of phangorn <a href="http://cran.r-project.org/web/packages/phangorn/vignettes/phangorn-specials.pdf">PDF</a>

<li>
Paper: <a href="http://bioinformatics.oxfordjournals.org/content/27/4/592">phangorn: phylogenetic analysis in R</a>

<li>
Presentation <a href="http://rug.mnhn.fr/semin-r/PDF/semin-R_phylogenetics_KSchliep_180310.pdf">PDF</a>

</ul>

<h2 id="toc_2.5">Other</h2>
<ul>
<li>
<a href="http://evolution.genetics.washington.edu/phylip.html">PHYLIP</a>

<li>
<a href="http://discover.nci.nih.gov/matchminer/index.jsp">MatchMiner</a> is a set of tools that enables the user to translate between disparate ids for the same gene.

<li>
<a href="http://main.g2.bx.psu.edu/">Galaxy</a>

<li>
<a href="http://genometools.org/index.html">GenomeTools: versatile open source genome analysis software</a>

</ul>

<h1 id="toc_3">Topic</h1>
<ul>
<li>
<a href="RNA-Seq.html">RNA-Seq</a>

<li>
<a href="samtools.html">samtools</a>

</ul>

	</div><!-- end of post -->
	<div class="backhome">
		<a href="../index.html">Homepage</a><br>
        <a href="#disqus_thread" data-disqus_identifier="vimwiki: Bioinformatics" id="disqus_thread-show" class="showLink" onclick="showHide('disqus_thread');return true;">Comments</a>
	</div>
<br>

<!-- javascript to hide comments -->
<script language="javascript" type="text/javascript">
/*** JavaScript: Show/Hide Content ***/
function showHide(shID) {
   if (document.getElementById(shID)) {
      if (document.getElementById(shID+'-show').style.display != 'none') {
         document.getElementById(shID+'-show').style.display = 'none';
         document.getElementById(shID).style.display = 'block';
      }
      else {
         document.getElementById(shID+'-show').style.display = 'inline';
         document.getElementById(shID).style.display = 'none';
      }
   }
}
</script>

<!-- javascript to show count of comments -->
<script type="text/javascript">
    /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
    var disqus_shortname = 'augix-wiki'; // required: replace example with your forum shortname

    /* * * DON'T EDIT BELOW THIS LINE * * */
    (function () {
        var s = document.createElement('script'); s.async = true;
        s.type = 'text/javascript';
        s.src = 'http://' + disqus_shortname + '.disqus.com/count.js';
        (document.getElementsByTagName('HEAD')[0] || document.getElementsByTagName('BODY')[0]).appendChild(s);
    }());
</script>


<a name="disqus_thread"></a>
<div id="disqus_thread" class="comment"><p align='right'><a href="#top" id="disqus_thread-hide" class="hideLink" onclick="showHide('disqus_thread');return true;">Hide Comments</a></p>
</div>

<script type="text/javascript">
    /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
    var disqus_shortname = 'augix-wiki'; // required: replace example with your forum shortname

    // The following are highly recommended additional parameters. Remove the slashes in front to use.
    // var disqus_identifier = 'unique_dynamic_id_1234';
    var disqus_identifier = "vimwiki: Bioinformatics";
    // var disqus_url = 'http://example.com/permalink-to-page.html';

    /* * * DON'T EDIT BELOW THIS LINE * * */
    (function() {
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
</script>
<noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>

</div><!-- end of container -->

</body>
</html>
