'''Chargaff's rules''' state that [[DNA]] from any cell of all organisms should have a 1:1 ratio of [[pyrimidine]] and [[purine]] bases and, more specifically, that the amount of [[guanine]] is equal to [[cytosine]] and the amount of [[adenine]] is equal to [[thymine]]. This pattern is found in both strands of the DNA. They were discovered by Austrian chemist [[Erwin Chargaff]].<ref name=Elson1952>Elson D, Chargaff E (1952). On the desoxyribonucleic acid content of sea urchin gametes. ''Experientia, 8(4):143-5''.</ref><ref name=Chargaff1952>Chargaff E, Lipshitz R, Green C (1952). Composition of the desoxypentose nucleic acids of four genera of sea-urchin. ''J Biol Chem, 195(1):155-60''.</ref><ref name=Chargaff1951a>Chargaff E, Lipshitz R, Green C, Hodes ME (1951). The composition of the deoxyribonucleic acid of salmon sperm. ''J Biol Chem, 192(1):223-30''.</ref><ref name=Chargaff1951b>Chargaff E (1951).Some recent studies on the composition and structure of nucleic acids. ''J Cell Physiol Suppl, 38(Suppl. 1):41-59''.</ref><ref name=Magasanik1950>Magasanik B, Vischer E, Doniger R, Elson D, Chargaff E (1950). The separation and estimation of ribonucleotides in minute quantities. ''J Biol Chem, 186(1):37-50.</ref><ref name=Chargaff>Chargaff E (1950). Chemical specificity of nucleic acids and mechanism of their enzymatic degradation. ''Experientia, 6(6):201-9''.</ref>

==Chargaff Parity Rule 1==
The first rule holds that a double-stranded DNA molecule globally %A = %T and %G = %C.<ref name=Chargaff>Chargaff E (1950). Chemical specificity of nucleic acids and mechanism of their enzymatic degradation. ''Experientia, 6(6):201-9''.</ref> The rigorous validation of the rule constitutes the basis of Watson-Crick pairs in the DNA double helix.

==Chargaff Parity Rule 2==
The second rule holds that globally both %A ~ %T and %G ~ %C are valid for each of the two DNA strands.<ref name=Rudner1968>Rudner R, Karkas JD, Chargaff E (1968). Separation of ''B. subtilis'' DNA into complementary strands. III. Direct Analysis. ''Proc Natl Acad Sci USA, 60:921-922''.</ref> This describes only a global feature of the base composition in a single DNA strand.<ref name=Zhang2003>Zhang CT, Zhang R, Ou HY (2003). The Z curve database: a graphic representation of genome sequences.</ref>

==Research==

The second of Chargaff's rules (or "Chargaff's second parity rule") is that the composition of DNA varies from one species to another; in particular in the relative amounts of A, G, T, and C bases. Such evidence of molecular diversity, which had been presumed absent from DNA, made DNA a more credible candidate for the genetic material than protein.

In 2006 it was shown that this rule applies to four of the five types of double stranded genomes; specifically it applies to the [[eukaryote|eukaryotic]] [[chromosomes]], the [[bacteria]]l chromosomes, the double stranded [[DNA]] viral genomes, and the [[archea]]l chromosomes.<ref>Mitchell D, Bridge R (2006). A test of Chargaff's second rule. ''Biochem Biophys Res Commun, 340(1):90-94''.</ref> It does not apply to the organellar genomes ([[mitochondria]] and [[plastid]]s) nor does it apply to the single stranded DNA (viral) genomes or any type of [[RNA]] genome. The basis for this rule is still under investigation. 

The rule itself has consequences. In most bacterial genomes (which are generally 80-90% coding) genes are arranged in such a fashion that approximately 50% of the coding sequence lies on either strand. [[Szybalski]], in the 1960s, showed that in [[bacteriophage]] coding sequences [[purines]] (A and G) exceed [[pyrimidines]] (C and T).<ref name=Szybalski1966>Szybalski W, Kubinski H, Sheldrick O (1966). Pyrimidine clusters on the transcribing strand of DNA and their possible role in the initiation of RNA synthesis. ''Cold Spring Harbor Symp Quant Biol, 31:123-127''.</ref> This rule has since been confirmed in other organisms and should probably be now termed "Szybalski's rule". While Sybalski's rule generally holds, exceptions are known to exist.<ref name=Cristillo1998>Cristillo AR (1998). "Characterization of G0/G1 switch genes in cultured T lymphocytes". PhD thesis. Queen's University, Kingston, Ontario Canada.</ref><ref name=Bell1999>Bell SJ, Forsdyke DR (1999). Deviations from Chargaff's second parity rule correlate with direction of transcription. ''J Theor Biol, 197:63-76''.</ref><ref name=Lao2000>Lao PJ, Forsdyke DR (2000). Thermophilic Bacteria Strictly Obey Szybalski's Transcription Direction Rule and Politely Purine-Load RNAs with Both Adenine and Guanine. ''Genome, 10(2):228-236''.</ref>  The biological basis for Szybalski's rule, like Chargaff's, is not yet known. 

The combined effect of Chargaff's second rule and Sybalski's rule can be seen in bacterial genomes where the coding sequences are not equally distributed. The [[genetic code]] has 64 [[codons]] of which 3 function as termination codons: there are only 20 amino acids normally present in proteins. (There are two uncommon [[amino acid]]s - [[selenocysteine]] and [[pyrolysine]] - found in a limited number of proteins and encoded by the 'stop' codons - TGA and TAG respectively.) The mismatch between the number of codons and amino acids allows several codons to code for a single amino acid. These codons normally differ in the third codon base position. 

Multivariate statistical analysis of codon use within genomes with unequal quantities of coding sequences on the two strands has shown that codon use in the third position depends on the strand on which the gene is located. This seems likely to be the result of Szybalski's and Chargaff's rules. Because of the asymmetry in pyrimidine and purine use in coding sequences, the strand with the greater coding content will tend to have the greater number of purine bases (Szybalski's rule). Because the number of purine bases will to a very good approximation equal the number of their complementary pyrimidines within the same strand and because the coding sequences occupy 80-90% of the strand, there appears to be (1) a selective pressure on the third base to minimise the number of purine bases in the strand with the greater coding content and (2) that this pressure is proportional to the mismatch in the length of the coding sequences between the two strands. 

The origin of the deviation from Chargaff's rule in the organelles has been suggested to be a consequence of the mechanism of replication.<ref name=Nikolaou2006>Nikolaou C, Almirantis Y (2006). Deviations from Chargaff's second parity rule in organellar DNA. Insights into the evolution of organellar genomes. ''Gene, 381:34-41''.</ref> During replication the DNA strands separate. In single stranded DNA, [[cytosine]] spontaneously slowly deaminates to [[adenosine]] (a C to A [[transversion]]). The longer the strands are separated the greater the quantity of deamination. For reasons that are not yet clear the strands tend to exist longer in single form in mitochondria than in chromsomal DNA. This process tends to yield one strand that is enriched in [[guanine]] (G) and [[thymine]] (T) with its complement enriched in cytosine (C) and adenosine (A) and this process may have given rise to the deviations found in the mitochondria.

Chargaff's second rule appears to be the consequence of a more complex parity rule: within a single strand of DNA any oligonucleotide is present in equal numbers to its complementary nucleotide. Because of the computational requirements this has not been verified in all genomes for all oligonucleotides. It has been verified for triplet oligonucleotides for a large data set.<ref name=Albrecht-Buehler2006>Albrecht-Buehler G (2006). Asymptotically increasing compliance of genomes with Chargaff's second parity rules through inversions and inverted transpositions. ''Proc Natl Acad Sci USA 103 (47) 17828-17833'' </ref> Albrecht-Buehler has suggested that this rule is the consequence of genomes evolving by a process of inversion and transposition. This process does not appear to have acted on the mitochondrial genomes.

==Relative proportions (%) of bases in DNA==
Both of Chargaff's rules are supported by the following table:{{Fact|date=February 2007}}
{| border="0" align="center" style="border: 1px solid #999; background-color:#FFFFFF"
|-align="left" bgcolor="#CCCCCC"
! Organism
!align="center"|A
!align="center"|T
!align="center"|C
!align="center"|G
|- 
| Human
| 30.9
| 29.4
| 19.9
| 19.8
|--bgcolor="#EFEFEF"
| Chicken
| 28.8
| 29.2
| 20.5
| 21.7
|-
| Grasshopper
| 29.3
| 29.3
| 20.5
| 20.7
|--bgcolor="#EFEFEF"
| Sea Urchin
| 32.8
| 32.1
| 17.7
| 17.3
|-
| Wheat
| 27.3
| 27.1
| 22.7
| 22.8
|--bgcolor="#EFEFEF"
| Yeast
| 31.3
| 32.9
| 18.7
| 17.1
|-
| ''E. Coli''
| 24.7
| 23.6
| 26.0
| 25.7
|}

==See also==
* [http://www.cbs.dtu.dk/services/GenomeAtlas/ CBS Genome Atlas Database] &mdash; contains hundreds of examples of base skews.<ref name=Hallin2004>Hallin PF, David Ussery D (2004). CBS Genome Atlas Database: A dynamic storage for bioinformatic results and sequence data. ''Bioinformatics, 20:3682-3686''.</ref>
* [http://tubic.tju.edu.cn/zcurve/ The Z curve database of genomes] &mdash; a 3-dimensional visualization and analysis tool of genomes.<ref name=Zhang2003>Zhang CT, Zhang R, Ou HY (2003). The Z curve database: a graphic representation of genome sequences. ''Bioinformatics, 19(5):593-599''.</ref>

==References==
<references/>
===Further reading===
* Szybalski W, Kubinski H, Sheldrick P (1966). Pyrimidine clusters on the transcribing strands of DNA and their possible role in the initiation of RNA synthesis. ''Cold Spring Harbor NY Symp. Quant Biol, 31:123â€“127''.
* Lobry JR (1996). Asymmetric substitution patterns in the two DNA strands of bacteria. ''Mol Biol Evol, 13:660-665''.
* Lafay B, Lloyd AT, McLean MJ, Devine KM, Sharp PM, Wolfe KH (1999). Proteome composition and codon usage in spirochaetes: species-specific and DNA strand-specific mutational biases. ''Nucleic Acids Res, 27:1642-1649''.
* McLean MJ, Wolfe KH, Devine KM (1998). Base composition skews, replication orientation, and gene orientation in 12 prokaryote genomes. ''J Mol Evol, 47:691-696''.
* McInerney JO. Replicational and transcriptional selection on codon usage in Borrelia burgdorferi. ''Proc Natl Acad Sci USA, 95:10698-10703''.

[[Category:DNA]]

[[fr:Coefficient de Chargaff]]
[[it:Regole di Chargaff]]
[[ru:ÐŸÑ€Ð°Ð²Ð¸Ð»Ð° Ð§Ð°Ñ€Ð³Ð°Ñ„Ñ„Ð°]]
[[uk:ÐŸÑ€Ð°Ð²Ð¸Ð»Ð° Ð§Ð°Ñ€Ð³Ð°Ñ„Ñ„Ð°]]