<html>
<head>
	<link rel="Stylesheet" type="text/css" href="../templates/skinny_disqus.css" />
	<title>Prepare sequence for PCR verification</title>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
</head>

<body>
<a name='top'></a>
<div class="container">
    <div class="header">
		<h1>Prepare sequence for PCR verification</h1> 
	</div><!-- end of header -->
	<div class="post"><!-- start of post -->
	
<p>
Get Consensus Sequence for Primer Design
</p>

<p>
Our task is to get the consensus sequence for primer design. The steps in this task are:
Note: all the input and output files should be in the fold "output" and all the perl scripts are in the fold "perl scripts".
</p>

<h1 id="toc_1">Get the information about exons (outlier &amp; detected) to form the Input File.</h1>
<p>
The information includes:
</p>
<ol>
<li>
the refseq ID of the human transcript in which the exons are located;  

<li>
the name of the chromosome in which the exons are located;

<li>
the chain of chromosome (+/-)

<li>
the ID of the outlier exon;

<li>
the ID and the start/stop position in the chromosome of the exons(including one outlier and other detected exons).  

</ol>

<p>
   Sample:
   REF_ID NM_000350
   CHR	chr1
   CHAIN	-
   OUTLIER	66741
   EXON_CLUSTER_ID START STOP
   66732 94230989 94231380
   66735 94234253 94234339
   66737 94236005 94236083   
   66738 94236107 94236254
   ...
</p>

<p>
You can get the positions of the exons from files named &lt;code&gt;&lt;chromosome name&gt;_exon_location.txt&lt;/code&gt;.
These files are produced using command: 
  perl get_exon_id_from_affymatrix.pl &lt;chromosome name&gt;
</p>

<h1 id="toc_2">Extract the human transcript sequence from Human Refseq Database and find the corresponding positions of the exons in the transcript.</h1>
<p>
run perl command:
  perl get_human_refseq_exon_position.pl &lt;Input File&gt;
Two files will be produced: &lt;code&gt;&lt;Refseq ID&gt;_seq.fa &lt;/code&gt; and &lt;code&gt;&lt;Refseq ID&gt;_location.txt&lt;/code&gt;. You should put the two files in the fold &lt;code&gt;output&lt;/code&gt;.
</p>

<h1 id="toc_3">Search the chimp and rhesus Refseq Database with human transcript using BLAST.</h1>
<p>
Commands:
 export PATH=$PATH:/usr/local/blastall
 blastall -p blastn -d ../rna_database/rna_chimp.fa -i  &lt;Refseq ID&gt;_seq.fa -o &lt;Refseq ID&gt;_chimp.txt -m8 -b1
 blastall -p blastn -d ../rna_database/rna_rhesus -i &lt;Refseq ID&gt;_seq.fa -o &lt;Refseq ID&gt;_rhesus.txt -m8 -b1
Two files will be produced: &lt;code&gt;&lt;Refseq ID&gt;_chimp.txt&lt;/code&gt; and  &lt;code&gt;&lt;Refseq ID&gt;_rhesus.txt&lt;/code&gt;.
</p>

<p>
ps: create blast database before lauching blastall
 formatdb -p F -o F -a F -i ../rna_database/rna_human.fa
 formatdb -p F -o F -a F -i ../rna_database/rna_chimp.fa
 formatdb -p F -o F -a F -i ../rna_database/rna_rhesus.fa
</p>

<h1 id="toc_4">Extract the chimp and rhesus transcript sequences from Chimp Refseq Database and Rhesus Refseq Database respectively.</h1>
<p>
Commands:
If you want to align human and chimp transcrips, run this:
 perl get_seqs_for_2alignment.pl &lt;Refseq ID&gt;
If you wan to align three species (human, chimp and rhesus), run this:
 perl get_seqs_for_3alignment.pl &lt;Refseq ID&gt;
A file will be produced: &lt;code&gt;&lt;Refseq ID&gt;_for_2alignment.fa&lt;/code&gt; or &lt;code&gt;&lt;Refseq ID&gt;_for_3alignment.fa&lt;/code&gt;.
</p>

<h1 id="toc_5">Align the transcripts of the species using Soft clustalw.</h1>
<p>
The Soft &lt;code&gt;clustalw&lt;/code&gt; is in the folder &lt;code&gt;clustalw1.83&lt;/code&gt;. The input file is  &lt;code&gt;&lt;Refseq ID&gt;_for_2alignment.fa&lt;/code&gt; or &lt;code&gt;&lt;Refseq ID&gt;_for_3alignment.fa&lt;/code&gt;.
Two file will be produced: &lt;code&gt;&lt;Refseq ID&gt;_for_2alignment.aln&lt;/code&gt; or &lt;code&gt;&lt;Refseq ID&gt;_for_3alignment.aln&lt;/code&gt; and &lt;code&gt;&lt;Refseq ID&gt;_for_2alignment.dnd&lt;/code&gt; or &lt;code&gt;&lt;Refseq ID&gt;_for_2alignment.dnd&lt;/code&gt;.
You should put the first file in fold &lt;code&gt;output&lt;/code&gt;.
</p>

<p>
 sh clustalw_for_2seq.sh &lt;Refseq ID&gt;
or 
 sh clustalw_for_3seq.sh &lt;Refseq ID&gt;
</p>

<h1 id="toc_6">Get the consensus sequence.</h1>
<p>
Command:
  perl get_consensus_seq.pl &lt;Refseq ID&gt; &lt;2/3&gt;
A file will be produced: &lt;code&gt;cons_2_&lt;Refseq ID&gt;.txt&lt;/code&gt; or &lt;code&gt;cons_3_&lt;Refseq ID&gt;.txt&lt;/code&gt;.
Note: the sequence of the outlier exon will be in capital characters and the start/end of the detected exon sequences will also be in capital characters.
</p>

<h1 id="toc_7">If you want to allow gaps in the consensus sequence, you should recalculate the positions of the exons.</h1>
<p>
Command:
 perl get_gapped_position.pl &lt;Refseq ID&gt; &lt;2/3&gt;
A file will be produced: &lt;code&gt;gapped_&lt;Refseq ID&gt;_location.txt&lt;/code&gt;.
</p>

<h1 id="toc_8">Files</h1>
<p>
*<a href="get_exon_id_from_affymatrix.pl">get_exon_id_from_affymatrix.pl</a>
*<a href="get_detected_exon.R">get_detected_exon.R</a>
</p>

	</div><!-- end of post -->
	<div class="backhome">
		<a href="../index.html">Main Page</a><br>
        <a href="#disqus_thread" data-disqus_identifier="vimwiki: Prepare sequence for PCR verification" id="disqus_thread-show" class="showLink" onclick="showHide('disqus_thread');return true;">Comments</a>
	</div>
<br>

<!-- javascript to hide comments -->
<script language="javascript" type="text/javascript">
/*** JavaScript: Show/Hide Content ***/
function showHide(shID) {
   if (document.getElementById(shID)) {
      if (document.getElementById(shID+'-show').style.display != 'none') {
         document.getElementById(shID+'-show').style.display = 'none';
         document.getElementById(shID).style.display = 'block';
      }
      else {
         document.getElementById(shID+'-show').style.display = 'inline';
         document.getElementById(shID).style.display = 'none';
      }
   }
}
</script>

<!-- javascript to show count of comments -->
<script type="text/javascript">
    /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
    var disqus_shortname = 'augix-wiki'; // required: replace example with your forum shortname

    /* * * DON'T EDIT BELOW THIS LINE * * */
    (function () {
        var s = document.createElement('script'); s.async = true;
        s.type = 'text/javascript';
        s.src = 'http://' + disqus_shortname + '.disqus.com/count.js';
        (document.getElementsByTagName('HEAD')[0] || document.getElementsByTagName('BODY')[0]).appendChild(s);
    }());
</script>


<a name="disqus_thread"></a>
<div id="disqus_thread" class="comment"><p align='right'><a href="#top" id="disqus_thread-hide" class="hideLink" onclick="showHide('disqus_thread');return true;">Hide Comments</a></p>
</div>

<script type="text/javascript">
    /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
    var disqus_shortname = 'augix-wiki'; // required: replace example with your forum shortname

    // The following are highly recommended additional parameters. Remove the slashes in front to use.
    // var disqus_identifier = 'unique_dynamic_id_1234';
    var disqus_identifier = "vimwiki: Prepare sequence for PCR verification";
    // var disqus_url = 'http://example.com/permalink-to-page.html';

    /* * * DON'T EDIT BELOW THIS LINE * * */
    (function() {
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        dsq.src = 'http://' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
</script>
<noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>

</div><!-- end of container -->

</body>
</html>
